ppl/cmd/zar/README.md

```diff
@@ -32,26 +32,24 @@ ln -s zq-sample-data/zng
 
 ## ingesting the data
 
-Let's take those logs and ingest them into a directory.   We'll make it
-easier to run all the commands by setting an environment variable pointing
-to the root of the logs tree.
+Let's take those logs and ingest them into a directory.  Often we'd keep our
+archive somewhere like [Amazon S3](https://aws.amazon.com/s3/), but since we're
+just doing a quick test we'll use local temp space.  We'll make it easier to
+run all the commands by setting an environment variable pointing to the root of
+the logs tree.
 ```
-mkdir ./logs
-set ZAR_ROOT=`pwd`/logs
+mkdir /tmp/logs
+ZAR_ROOT=/tmp/logs
 ```
 
 Now, let's ingest the data using "zar import".  We are working on more
 sophisticated ways to ingest data (e.g., by the standard time partitioning
 techniques of year/month/day/hour etc), but for now zar import just chops
-its input into chunk files of approximately equal size.
-
-Zar import expects its input to be
-in the zng format so we'll use zq to take all the zng logs, gunzip them,
-and feed them to "zar import", which here expects its data on stdin.  We'll chop them
-into chunks of 25MB, which is very small, but in this example the data set is
+its input into chunk files of approximately equal size. We'll chop into chunks
```

Comment:
I expect this wording needs to be updated, since we do at least _some_ time partitioning and sort the data by time, whereas this text looks like an artifact from when we did none of that. I'd appreciate it if someone could make a suggestion with the most accurate summary of how sophisticated (or not) it is currently.

Reply:
Coming soon... arbitrary partition keys and auto-sizing of partitions.